capsul/pipeline/pipeline.py

```diff
@@ -1038,6 +1038,7 @@ def export_parameter(self, node_name, plug_name,
                     pipeline_parameter))
 
         f = field(type_=source_field)
+        f.name = pipeline_parameter
 
```

Comment:
Name could be given as a `field()` parameter.

Reply:
oh yes, I'm not used to field() yet. I'm changing that.